src/components/slider/slider.tsx

```diff
@@ -124,16 +124,49 @@ export class Slider {
             return;
         }
 
+        const inputElement: HTMLInputElement = element.querySelector('input');
+        const value = this.getValue();
+
         /*
         For some reason the input element's `value` attribute is removed
         (probably by Stencil) when the element is first rendered. But if the
         attribute is missing when MDCSlider is initialized (MDC v11.0.0),
         MDCSlider crashes.
         So we add the attribute right before initializing MDCSlider. /Ads
         */
-        element
-            .querySelector('input')
-            .setAttribute('value', `${this.multiplyByFactor(this.getValue())}`);
+        inputElement.setAttribute('value', `${this.multiplyByFactor(value)}`);
+
+        /*
+        When creating the `mdcSlider` component, its important that the value set in
+        the input field obeys the range and the step size.
+
+        The MDCSlider will throw an exception unless the value in the input element
+        is dividible by the step value and is in the provided range.
+        If an exception occurs, this component will crash and it will be impossible to change
+        its value.
+        The logic below ensures that the component will render even though the
+        provided value is wrong.
+        This could be considered wrong, but it at least fixes so that it's possible
+        to change the value from the UI.
+        */
+        const greaterThanOrEqualMin = value >= this.valuemin;
+        const lessThanOrEqualMax = value <= this.valuemax;
+        const dividableByStep = value % this.step === 0;
+        const delta = 0.01;
+
+        if (!greaterThanOrEqualMin) {
+            const newMin = this.multiplyByFactor(value) - delta;
```

Comment:
```suggestion
            const newMin = this.multiplyByFactor(value);
```

Just remove delta, I don't see the use here